python/cuspatial/cuspatial/core/binpreds/feature_covers.py

```diff
@@ -36,21 +49,58 @@ class CoversPredicateBase(EqualsPredicateBase):
     pass
 
 
+class LineStringLineStringCovers(BinPred):
+    def _preprocess(self, lhs, rhs):
+        return _basic_equals_all(rhs, lhs)
+
+
+class PolygonPointCovers(BinPred):
+    def _preprocess(self, lhs, rhs):
+        return _basic_contains_any(lhs, rhs)
```

Comment:
Should this be `contains_all`? All points must be in the polygon => polygon covers point.

Reply:
In this case any is equivalent to all since the `rhs` geometry are points. I think you're right that I'll want `all` when I implement multipoints, but at this stage `_basic_contains_all` hasn't been implemented. I'll worry about that when I get to multipoints if that works for you.